software/lidatube.yml

```
@@ -0,0 +1,11 @@
+name: "LidaTube"
+website_url: "https://github.com/TheWicklowWolf/LidaTube"
+source_code_url: "https://github.com/TheWicklowWolf/LidaTube"
+description: "A self-hosted video downloader and media manager with a clean interface. Rip audio from YT based on your Lidarr wanted list."
```

```suggestion
description: "Finding and fetch missing Lidarr albums via yt-dlp."
```

More accurate description (and also the official one)

---

They must have updated it soon after I added this, because I swear I just copy-pasted the description from their project and added the second sentence. A definite improvement.